src/DialpadPlugin.js

```diff
@@ -23,14 +21,4 @@ export default class DialpadPlugin extends FlexPlugin {
 
     this.registerReducers(manager);
```

Comment:
Need to remove this line of code. Plugin is currently throwing an error here and failing to load.

```
twilio-flex.unbundled-react.min.js:1476 6/29/2021, 2:16:37 PM - PluginManager: There was an error during 'flex-dialpad-addon-plugin' plugin initialization

Original error:
"this.registerReducers is not a function"
```

Reply:
Done!